### ISSUE TITLE
Extend address profile to include phone number and any other related contact information

### ISSUE THREAD

**Issue by siuc-nate** [CredentialTransparencyInitiative/vocabularies#135]

**Comment by stuartasutton** [comment]
@siuc-nate can you be more specific. Phone number and what else--**exactly**. Don't say anything else because there's a lot that probably cannot be justified.

**Comment by siuc-nate** [comment]
This issue was added as @jkitchensSIUC and I were in the midst of reviewing a bunch of CTDL stuff - I believe her intent was to enable associating a phone number with an address, so that a user would be able to call a given physical location directly.  I could see several potential things that could be tied to addresses (ContactPoint, Email, Fax, etc.) but those things might _also_ apply at the organization level (ie not tied to a location), and I'm not sure how we'd handle that (aside from just having those kinds of properties in both places in the schema).

In other words, @jkitchensSIUC is more qualified to give you a thorough answer on this than I am.

**Comment by stuartasutton** [comment]
:-) I was not concerned about the phone number as much as an open-ended "any other related contact information". Note that the address property references a thing -- [PostalAddress] (http://schema.org/PostalAddress) that has it all. So one could have an address and an array of PostalAddress with each being a different place with all the info needed.

**Comment by jkitchensSIUC** [comment]
@stuartasutton @siuc-nate we need to update the spreadsheet to include we need to add phone numbers and emails to the address with the PostalAddress. Also, need to determine the guidance for when to associate email and phone number with an organization or an address. @stuartasutton will you update the spreadsheet? Or we can put on agenda for tomorrow.

**Comment by stuartasutton** [comment]
Telephone and faxNumber have been already added to PostalAddress. PostAddress includes the entity ContactPoint which has a telephone/fax numbers as well as other contact properties that might be considered.

**Comment by jkitchensSIUC** [comment]
@siuc-nate @mparsons1953 what else needs to be done with this issue?  Is the issue now updating the editor? Are there other properties needed?

**Comment by siuc-nate** [comment]
The problem we need to tackle is how to handle cases where information like phone number or email address is tied to an address vs cases where that information is universal for the entire organization.

**Comment by stuartasutton** [comment]
I believe all the properties are present in PostalAddress (name, telephone, faxNumber).

Schema.org n general does not help since Organization supports telephone etc. directly as well as address==>PostalAddress where you have them again. Currently, with the exception of name, our CredentialOrganization puts it all in PostalAddress...which seems to make your concern mute, Nate...or am I nt getting it.

**Comment by siuc-nate** [comment]
http://credreg.net/page/domainsviewer#PostalAddress
This profile needs:
- name
- email

**Comment by stuartasutton** [comment]
**ACTION TAKEN**

Properties added:

> [ctdl:email](http://www.credreg.net/page/propertiesviewer#email)
> [ctdl:name](http://www.credreg.net/page/propertiesviewer#name)

Closing issue.
